packages/components/steps/src/item.vue

```diff
@@ -87,13 +86,18 @@ export interface IStepsInject {
 
 export default defineComponent({
   name: 'ElStep',
+  components: {
+    ElIcon,
+    Close,
+    Check,
+  },
   props: {
     title: {
       type: String,
       default: '',
     },
     icon: {
-      type: String,
+      type: [String, Object] as PropType<string | Component>,
```

Comment:
Please use `buildProp`

Reply:
How to define `Component` type?
Like this?

```js
type: [String, Object as Component] 
```

Reply:
Forget it, I will modify it in my PR.